src/pages/divisions.js

```diff
@@ -14,17 +14,20 @@ function list(templates, division, req, res) {
   })
 }
 
-function nav(templates, season, division, req, res) {
+function nav(templates, season, division, admin, req, res) {
   var division_id = req.params.division_id
   division.getDivision(division_id).then(division => {
-    return season.getActiveSeason().then(seasons => {
-      var html = templates.division.division({
-        user: req.user,
-        division: division,
-        seasons: seasons
+    admin.getDivisionAdmins(division_id).then(divisionAdmins => {
```

Comment:
Missing `return` on this promise. Not returning promises causes timeouts on errors instead of actually printing it out and returning a 500.